From 506973e0f4b070731c3be463d7f496805a74989a Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Thu, 17 Sep 2015 12:46:46 -0500
Subject: [PATCH 60/86] arch/arm64: Axxia Device Tree Updates

Remove references to mmc in simulation, as it is no longer
supported.

Signed-off-by: John Jacques <john.jacques@intel.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/intel/axm5604-sim.dts |   13 -------------
 arch/arm64/boot/dts/intel/axm5608-sim.dts |   13 -------------
 arch/arm64/boot/dts/intel/axm5616-sim.dts |   13 -------------
 3 files changed, 0 insertions(+), 39 deletions(-)

diff --git a/arch/arm64/boot/dts/intel/axm5604-sim.dts b/arch/arm64/boot/dts/intel/axm5604-sim.dts
index 6dbfa70..969cf5b 100644
--- a/arch/arm64/boot/dts/intel/axm5604-sim.dts
+++ b/arch/arm64/boot/dts/intel/axm5604-sim.dts
@@ -53,19 +53,6 @@
 			smsc,irq-active-high;
 			smsc,irq-push-pull;
 		};
-
-		amba {
-			mmci@8021030000 {
-				compatible = "arm,pl18x", "arm,primecell";
-				reg = <0x80 0x21030000 0x10000>;
-				interrupts = <GIC_SPI 281 IRQ_TYPE_LEVEL_HIGH>,
-					     <GIC_SPI 282 IRQ_TYPE_LEVEL_HIGH>;
-				vmmc-supply = <&mmc_3v3>;
-				vqmmc-supply = <&mmc_3v3>;
-				clocks = <&clks 0>, <&clks 1>;
-				clock-names = "apb_pclk", "mmc_clk";
-			};
-		};
 	};
 };
 
diff --git a/arch/arm64/boot/dts/intel/axm5608-sim.dts b/arch/arm64/boot/dts/intel/axm5608-sim.dts
index 8d1eb1f..6c533aa 100644
--- a/arch/arm64/boot/dts/intel/axm5608-sim.dts
+++ b/arch/arm64/boot/dts/intel/axm5608-sim.dts
@@ -53,19 +53,6 @@
 			smsc,irq-active-high;
 			smsc,irq-push-pull;
 		};
-
-		amba {
-			mmci@8021030000 {
-				compatible = "arm,pl18x", "arm,primecell";
-				reg = <0x80 0x21030000 0x10000>;
-				interrupts = <GIC_SPI 281 IRQ_TYPE_LEVEL_HIGH>,
-					     <GIC_SPI 282 IRQ_TYPE_LEVEL_HIGH>;
-				vmmc-supply = <&mmc_3v3>;
-				vqmmc-supply = <&mmc_3v3>;
-				clocks = <&clks 0>, <&clks 1>;
-				clock-names = "apb_pclk", "mmc_clk";
-			};
-		};
 	};
 };
 
diff --git a/arch/arm64/boot/dts/intel/axm5616-sim.dts b/arch/arm64/boot/dts/intel/axm5616-sim.dts
index c3a2dca..877e325 100644
--- a/arch/arm64/boot/dts/intel/axm5616-sim.dts
+++ b/arch/arm64/boot/dts/intel/axm5616-sim.dts
@@ -53,19 +53,6 @@
 			smsc,irq-active-high;
 			smsc,irq-push-pull;
 		};
-
-		amba {
-			mmci@8021030000 {
-				compatible = "arm,pl18x", "arm,primecell";
-				reg = <0x80 0x21030000 0x10000>;
-				interrupts = <GIC_SPI 281 IRQ_TYPE_LEVEL_HIGH>,
-					     <GIC_SPI 282 IRQ_TYPE_LEVEL_HIGH>;
-				vmmc-supply = <&mmc_3v3>;
-				vqmmc-supply = <&mmc_3v3>;
-				clocks = <&clks 0>, <&clks 1>;
-				clock-names = "apb_pclk", "mmc_clk";
-			};
-		};
 	};
 };
 
-- 
1.7.5.4

